src/parser/monk/windwalker/modules/spells/BlackoutKick.js

```diff
@@ -19,29 +19,29 @@ const COOLDOWN_REDUCTION_MS = 1000;
 class BlackoutKick extends Analyzer {
   static dependencies = {
     spellUsable: SpellUsable,
-    abilityTracker: AbilityTracker,
   };
 
   IMPORTANT_SPELLS = [
     SPELLS.RISING_SUN_KICK.id,
     SPELLS.FISTS_OF_FURY_CAST.id,
-    SPELLS.WHIRLING_DRAGON_PUNCH_TALENT.id,
   ];
 
+  constructor(...args) {
+    super(...args);
+
+    if (this.selectedCombatant.hasTalent(SPELLS.WHIRLING_DRAGON_PUNCH_TALENT.id)) {
+      this.IMPORTANT_SPELLS.push(SPELLS.WHIRLING_DRAGON_PUNCH.id);
```

Comment:
Shouldn't this be `SPELLS.WHIRLING_DRAGON_PUNCH_TALENT.id` not `SPELLS.WHIRLING_DRAGON_PUNCH.id`